packages/react-next/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -132,21 +104,24 @@ export class ChoiceGroupBase extends React.Component<IChoiceGroupProps, IChoiceG
             {options.map((option: IChoiceGroupOption) => {
               const innerOptionProps = {
                 ...option,
-                focused: option.key === keyFocused,
-                checked: option.key === keyChecked,
+                focused: option.optionKey === keyFocused,
+                checked: option.optionKey === keyChecked,
                 disabled: option.disabled || disabled,
-                id: this._getOptionId(option),
-                labelId: `${this._labelId}-${option.key}`,
-                name: name || this._id,
+                id: getOptionId(option, id),
+                labelId: `${choiceGroupLabelId}-${option.optionKey}`,
+                name: name || id,
                 required,
               };
 
               return (
                 <ChoiceGroupOption
-                  key={option.key}
-                  onBlur={this._onBlur}
-                  onFocus={this._onFocus(option.key)}
-                  onChange={this._onChange(option.key)}
+                  key={option.optionKey}
+                  // eslint-disable-next-line react/jsx-no-bind
```

Comment:
This is a case where I think the callbacks should be memoized, since they're being passed to another custom component.